<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PagePapersClass</class>
 <widget class="QWidget" name="PagePapersClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PagePapers</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitterHorizontal">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitterPapers">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="PaperTableView" name="tvPapers">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabDetails">
        <attribute name="title">
         <string>Detals</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QSplitter" name="splitterDetails">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="handleWidth">
            <number>5</number>
           </property>
           <widget class="QWidget" name="layoutWidget">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>2</number>
             </property>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Abstract</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="teAbstract">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>2</number>
             </property>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Note</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="teNote">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget_2">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>2</number>
             </property>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Attachments</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="AttachmentsWidget" name="widgetAttachments" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabSnippets">
        <attribute name="title">
         <string>Snippets</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="margin">
          <number>5</number>
         </property>
         <item>
          <widget class="SnippetsTableView" name="tvSnippets">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitterTags">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QListView" name="lvTags">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>All tags</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="margin">
         <number>5</number>
        </property>
        <property name="spacing">
         <number>5</number>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="btFilter">
          <property name="icon">
           <iconset resource="MainWindow.qrc">
            <normaloff>:/MainWindow/Images/Filter.png</normaloff>:/MainWindow/Images/Filter.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="btAddTagToPaper">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="icon">
           <iconset resource="MainWindow.qrc">
            <normaloff>:/MainWindow/Images/Up.png</normaloff>:/MainWindow/Images/Up.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="btDelTagFromPaper">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="icon">
           <iconset resource="MainWindow.qrc">
            <normaloff>:/MainWindow/Images/Down.png</normaloff>:/MainWindow/Images/Down.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="5">
         <widget class="QListView" name="lvAllTags">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="btRenameTag">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="icon">
           <iconset resource="MainWindow.qrc">
            <normaloff>:/MainWindow/Images/TagEdit.png</normaloff>:/MainWindow/Images/TagEdit.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btAddTag">
          <property name="icon">
           <iconset resource="MainWindow.qrc">
            <normaloff>:/MainWindow/Images/TagAdd.png</normaloff>:/MainWindow/Images/TagAdd.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>AttachmentsWidget</class>
   <extends>QWidget</extends>
   <header>AttachmentsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PaperTableView</class>
   <extends>QTableView</extends>
   <header>PaperTableView.h</header>
  </customwidget>
  <customwidget>
   <class>SnippetsTableView</class>
   <extends>QTableView</extends>
   <header>SnippetsTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
